src/mappings/JoinAndQuit/mapping.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  JoinInProposal,
+  ProposalExecuted,
+  RedeemReputation,
+} from '../../types/JoinAndQuit/JoinAndQuit';
+
```

Comment:
where is rageQuit is handled ?

Reply:
What should the subgraph do in a ragequit?

Reply:
index the event of that and for each funder decreased it funding.
does a ragequit member should get out of the dao is a good point.

Reply:
I will add that when you release the Arc v9 with the fix of the ragequit

Reply:
do you mean arc V11 ?